contents/docs/cdp/batch-exports/postgres.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Postgres destination for batch exports
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+Batch exports can be used to export data to a Postgres table.
+
+## Setting up Postgres access
+
+1. Make sure PostHog can access your Postgres database.
```

Comment:
How is this different than "Create a Postgres user with table creation privileges."? Can we just combine this two into one point.

Reply:
1. Refers to allowing a connection to the host of the Postgres database. This host may be a managed solution like AWS RDS, Azure SQL, or any server. 2. Refers to the permissions once we are connected to the host. They are different points because they are independent: We may have a connection to the host, but no database permissions, and vice-versa.

Reply:
Sounds good, thanks for clarifying